var/spack/repos/builtin/packages/py-pytest-cov/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyPytestCov(PythonPackage):
+    """Pytest plugin for measuring coverage."""
+
+    homepage = "https://github.com/pytest-dev/pytest-cov"
+
+    version('2.3.1', '8e7475454313a035d08f387ee6d725cb',
+            url='https://pypi.python.org/packages/4c/2f/de9e8f226d4fb86f330ee2fcc709204fbd3aeab6ce17756b3cf5ea9aa4d7/pytest-cov-2.3.1.tar.gz')
```

Comment:
Can you convert this to `pypi.io`? The URL is:
https://pypi.io/packages/source/p/pytest-cov/pytest-cov-2.3.1.tar.gz